diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0000000..1111111 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.16)

 project(plasma-panel-colorizer)

-find_package(Qt6 ${QT_MIN_VERSION} REQUIRED)
+find_package(Qt6 ${QT_MIN_VERSION} REQUIRED COMPONENTS Gui Qml)
 find_package(ECM ${KF6_MIN_VERSION} REQUIRED NO_MODULE)
 find_package(Plasma ${PLASMA_MIN_VERSION} REQUIRED)
